dask/array/tests/test_creation.py

```diff
@@ -1,26 +1,18 @@
 import numpy as np
+from numpy.testing import assert_array_almost_equal
+import pytest
 
-import dask
 import dask.array as da
 
 
 def eq(a, b):
-    if isinstance(a, da.Array):
-        adt = a._dtype
-        a = a.compute(get=dask.get)
-    if isinstance(b, da.Array):
-        bdt = b._dtype
-        b = b.compute(get=dask.get)
-    else:
-        bdt = getattr(b, 'dtype', None)
-
-    if not str(adt) == str(bdt):
-        return False
-
-    c = a == b
-    if isinstance(c, np.ndarray):
-        c = c.all()
-    return c
+    """a is a dask array, b is a numpy array. Checks dtype, shape and value. No
+    `assert` needed.
+    """
+    a = np.array(a)
+    assert a.shape == b.shape
+    assert a.dtype == b.dtype
+    assert_array_almost_equal(a, b)
```

Comment:
Perhaps the eventual merged `eq` function should behave like this instead of returning True/False